assets/js/components/ErrorHandler/index.js

```diff
@@ -30,6 +50,15 @@ class ErrorHandler extends Component {
 		this.setState( { error, info } );
 	}
 
+	onErrorClick( e ) {
+		const range = document.createRange();
+		range.selectNodeContents( e.target );
+
+		const selection = global.getSelection();
+		selection.removeAllRanges();
+		selection.addRange( range );
```

Comment:
Let's attach this handler to a new "Copy error to clipboard" button and copy the text to the clipboard when the user clicks it. We can use [`document.execCommand( 'copy' )`](https://caniuse.com/#feat=mdn-api_document_execcommand_copy) to do this which has very good browser support, and then deselect the text

We can also wrap the copied text in single backticks (like WP does with Site Health information) so that we get better formatting in the forum when users use this to report an error this way.